src/mount.ts

```diff
@@ -1,12 +1,12 @@
 import { registerHtml } from './dom-wrappers';
-import { Container, TramOneComponent } from './types';
+import { Container, RootTramOneComponent } from './types';
 
 /**
  * Updates a container with an initial component for the first render.
  * @param component the tram-one component to render
  * @param container an element to render the component on
  */
-export default (component: TramOneComponent, container: Container) => {
+export default (component: RootTramOneComponent, container: Container) => {
```

Comment:
updated here (and in start.ts) to use RootTramOneComponent - a variant of TramOneComponent with an empty props and no children (technically we could enable the user to pass in props / children in the future - which would be great for server-side rendering, but that's for a future PR)